src/main/java/com/textmessenger/controller/DialogController.java

```diff
@@ -26,13 +26,13 @@ public DialogController(DialogService dialogService) {
   }
 
   @PostMapping("/user/{id}")
-  public ResponseEntity createDialog(@PathVariable("id") Long user, @Valid @RequestBody UserRxDto mainUser) {
+  public ResponseEntity createDialog(@PathVariable("id") Long user, @Valid @RequestBody User mainUser) {
     dialogService.createdByUserDialogWithUser(mainUser.getLogin(), user);
     return Optional.of(ResponseEntity.ok()).orElse(ResponseEntity.unprocessableEntity()).build();
   }
 
   @GetMapping("/user/{id}")
-  public ResponseEntity<?> readDialog(@Valid @PathVariable("id") UserRxDto user) {
+  public ResponseEntity<?> readDialog(@Valid @PathVariable("id") User user) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove usage of generic wildcard type. ([squid:S1452](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS1452))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=158&issues=AWWU5_aeAJjEPbGfqJYy&open=AWWU5_aeAJjEPbGfqJYy)

[ISSUE_KEY]: AWWU5_aeAJjEPbGfqJYy